### PR TITLE
[pt] Added rule ID:PELO_QUAL_PELA_QUAL and rule ID:PELA_QUAL_PELO_QUAL

### PR DESCRIPTION
**Pull request by marcoagpinto**
Heya, Susana and Pedro,

Rules for gender agreement.

Number of hits against 900 000 sentences=0

Thanks!

### CODE REVIEW

**Comment by marcoagpinto** [comment]
Heya,

Give me a few more minutes.

I have improved the rule, and I am testing it.

**Comment by marcoagpinto** [comment]
Heya,

I have improved it a bit, still no hits in 900 000 sentences.

**Comment by marcoagpinto** [comment]
Please don't review it yet, I came up with an idea, and I am testing it.

Give me more 30 minutes or 1 hour.

Thanks!

**Comment by marcoagpinto** [comment]
Ahhhh… removed a duplicate example in an AP, how come `TESTRULES PT` didn't spot it?

File: languagetool-language-modules/pt/src/main/resources/org/languagetool/rules/pt/grammar.xml [hunk 1]
```diff
@@ -7811,6 +7811,67 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA.
                 <token postag='A.+' postag_regexp='yes'/>
                 <example>Deixei o local um pouco irritada.</example>
             </antipattern>
+
+            <rule id='PELO_QUAL_PELA_QUAL' name="Concordância Género: 'pelo' qual → 'pela'" default="temp_off">
```
Comment (susanaboatto) [inline languagetool-language-modules/pt/src/main/resources/org/languagetool/rules/pt/grammar.xml:7815]: Let's avoid dialect-biased spelling in the rule name. Better: `Concordância: 'pelo' → 'pela'`

File: languagetool-language-modules/pt/src/main/resources/org/languagetool/rules/pt/grammar.xml [hunk 2]
```diff
@@ -7811,6 +7811,67 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA.
...
+                </antipattern>
+                <pattern>
+                    <token postag='SPS00|(SPS00:)?[DP]..F.+' postag_regexp='yes'/> <!-- female, SPS00="a" -->
+                    <token postag='AQ.[CF].+|NC[CF].+' postag_regexp='yes'/>
+                    <marker>
+                        <token regexp='yes'>pelos?</token>
+                    </marker>
+                    <token regexp='yes'>qua(l|is)</token>
+                </pattern>
+                <message>Possível erro de concordância de género.</message>
```
Comment (susanaboatto) [inline languagetool-language-modules/pt/src/main/resources/org/languagetool/rules/pt/grammar.xml:7839]: Again, `género` is a pt-PT spelling. Better: `Possível erro de concordância. Verifique se não quis dizer:`

File: languagetool-language-modules/pt/src/main/resources/org/languagetool/rules/pt/grammar.xml [hunk 3]
```diff
@@ -7811,6 +7811,67 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA.
...
+                </antipattern>
+                <pattern>
+                    <token postag='(SPS00:)?[DP]..M.+' postag_regexp='yes'/> <!-- male -->
+                    <token postag='AQ.[CM].+|NC[CM].+' postag_regexp='yes'/>
+                    <marker>
+                        <token regexp='yes'>pelas?</token>
+                    </marker>
+                    <token regexp='yes'>qua(l|is)</token>
+                </pattern>
+                <message>Possível erro de concordância de género.</message>
```
Comment (susanaboatto) [inline languagetool-language-modules/pt/src/main/resources/org/languagetool/rules/pt/grammar.xml:7868]: Same here

**Comment by marcoagpinto** [comment]
Fixed!

**Pull request merged**